draft-packages/likert-scale-legacy/docs/LikertScaleLegacy.stories.tsx

```diff
@@ -48,15 +51,16 @@ const scale: Scale = [
   },
 ]
 
-const labelId = "456"
-
 export const DefaultStory = () => {
```

Comment:
```suggestion
export const DefaultStory = args => {
  // spread the args into the component
```

Worth fixing the controls while we're here :)